src/net.cpp

```diff
@@ -1826,10 +1826,10 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
     }
 
     // Initiate network connections
-    int64_t nStart = GetTime();
+    auto nStart = GetTime<std::chrono::seconds>();
 
     // Minimum time before next feeler connection (in microseconds).
-    int64_t nNextFeeler = PoissonNextSend(nStart*1000*1000, FEELER_INTERVAL);
+    int64_t nNextFeeler = PoissonNextSend(nStart.count()*1000*1000, FEELER_INTERVAL);
```

Comment:
Ah, I did see that. Wasn't sure I should lint fix a line I didn't make any other changes on as it would change the git blame output. What is the standard practice?